src/libraries/System.Private.CoreLib/src/ILLink/ILLink.Substitutions.Shared.xml

```diff
@@ -21,5 +21,8 @@
     <type fullname="System.StartupHookProvider" feature="System.StartupHookProvider.IsSupported" featurevalue="false">
       <method signature="System.Boolean get_IsSupported()" body="stub" value="false" />
     </type>
+    <type fullname="System.Threading.ThreadPool" feature="System.Threading.ThreadPool.EnableDispatchAutoreleasePool" featurevalue="false">
+      <method signature="System.Boolean get_EnableDispatchAutoreleasePool()" body="stub" value="false" />
```

Comment:
One thing that might cause problems is that on `Browser.Mono`, this property doesn't appear to exist. So if someone set the feature switch, I'm guessing the linker is going to raise an exception.

Reply:
I added this property on Browser.Mono to fix the wasm build.

Reply:
But it isn't a property on `Browser.Mono`. It is a const.

Reply:
Good point. I’ll change it to a property so the linker doesn’t freak out.